### PR TITLE
Started filling out strings for typescript

### LINKED ISSUE #429: Add TypeScript strings
# Description

Strings for TypeScript aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Requirements

* Copy `web/thesauruses/_meta/strings.json` to the right TypeScript directory in `web/thesauruses/`
* Edit the strings.json file to accurately reflect the latest major version of TypeScript
* Delete the `"name":` fields before your PR
* Add in `"comment":` if you wish to explain something in the code
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then delete the `"code"` line.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

### PR DESCRIPTION
**Pull request by blockingHD**
## What GitHub issue does this PR apply to?

Resolves #429 

## What changed and why?

Added typescript strings

## Checklist
- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: web/thesauruses/typescript/strings.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
+{
+  "meta": {
+		"language": "ts",
```
Comment (geekygirlsarah) [inline web/thesauruses/typescript/strings.json]: This should be "typescript" to match the conventions used elsewhere in CT.

File: web/thesauruses/typescript/strings.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+      "not-implemented": true
+    },
+    "length_of_string": {
+      "name": "Length of string",
+      "code": ["'four'.length;"]
+    },
+    "max_length_of_string": {
+      "name": "Maximum length of string",
+      "not-implemented": true,
+      "comment": "Depends on the browser being used."
```
Comment (geekygirlsarah) [inline web/thesauruses/typescript/strings.json:110]: Huh, I never knew this!

File: web/thesauruses/typescript/strings.json (hunk 1 above)
Comment (blockingHD) [reply web/thesauruses/typescript/strings.json]: Whoops, changed.

**Comment by blockingHD** [comment]
In vscode to get rid of the names after filling out a file I have been using find a replace with this regex: '"name": "(.)+",\n'.

**Comment by geekygirlsarah** [comment]
Oops, I think I missed the changes on this somehow. But it looks good! I'll merge it in. 

Thanks for continuing to help out with Code Thesaurus, I appreciate it!

**Pull request merged**